src/samples/FullPortal/index.tsx

```diff
@@ -13,7 +13,17 @@ declare const PCore: any;
 declare const myLoadPortal: any;
 declare const myLoadDefaultPortal: any;
 
+function useQuery() {
+  const { search } = useLocation();
+
+  return useMemo(() => new URLSearchParams(search), [search]);
+}
+
 export default function FullPortal() {
+  const query = useQuery().get('portal');
```

Comment:
can you just listen to custom hook useQuery() here and use `query.get('portal')` inside if condition?

Reply:
updated